less_users/tests/test_user_challenge.py

```diff
@@ -1,18 +1,97 @@
-from less_users.models import UserChallenge
+import pytest
 
+from freezegun import freeze_time
+from datetime import date
 
-def test_user_challenge_correct_creation(user, challenge_1_day, create_user_challenge):
-    user_challenge = create_user_challenge
+from less_users.models import UserChallenge, Log
+
+
+def test_user_challenge_correct_creation(
```

Comment:
ale w teście nie tworzysz obiektu - bierzesz go z fixture. Co w takim razie ten test sprawdza?

Reply:
> Czy zmienić np. na taki, który nie pozwoli utworzć user_challenge gdy user się nie zalogował?

Niezbyt rozumiem.

Jeżeli chcesz sprawdzić czy `UserChallenge` został utworzony to przetestuj widok, który ten challenge robi. To chyba najmniejsza funkcja napisana przez Ciebie, która tworzy obiekt.

Reply:
OK, zrobię test widoku, który tworzy UserChallenge.